databrowser/package-lock.json

```diff
@@ -1,7528 +1,8329 @@
 {
```

Comment:
File indentation seems odd, could you please reformat the file using the default 2 spaces?